### DISCUSSION TITLE
Jira integration and little thingy on Dtrack UI

### DISCUSSION THREAD

**Discussion by Najafov007** [DependencyTrack/dependency-track#4121] [Q&A]
Hello dear dependency Track community! Recently started to use dtrack in our company and now we are facing 1 major and 1 minor problem. 

1) Major is we are unable to connect and integrate it with Jira. What exactly we want is after the CI pipeline that generates and adds projects under their parent name ( Only parent -> child, not only child) to create a taks in jira that says you have those vulnrs in this project. However we created API Token and added to Dtrack - it didnt, work. Maybe you guys will tell us what to do step by step, will be appreciated

2) Minor is as I said in our CI pipeline after it done we are creating a parent project with group name of gitlab repo with no sbom file and child project under that parent with the name of the repo itself and sbom. However, visually its the problem like parent shows 0 risk score, but under that parent there for example 10 childs with their own risk scores. Is there any chance to make parent like show the summary of the risk score of child projs under them? 
Thank you so so much!

**Comment by swenp76** [comment]
I've got the the JIRA integration working right now, here is how:
`tail -f dependency-track.log` will help a lot for debugging.
In integrations:
Use your JIRA user (email address) and the API Token as the password, do **not** set the "Authenticate using your personal API token" as this will result in "403" errors. 
Use `https://xxxx.atlassian.net/` as Base-URL. "rest/api/2/issue" will be attached automatically.
In alarms:
Tick the "Log successful alarms" for debugging purposes.
Enter the jira project key in 'target'.
Enter the issue type in 'ticket type', e.g. 'Task'.
Don't forget to set it to 'Activate'.
When the alarm is triggered and was successful you'll see a `Destination acknowledged reception of notification with status code 201` in the log.

**Reply by whiteninja76** [reply]
Is this still a valid way to set this up in the UI ?
